app/src/main/java/de/hhn/aib/labsw/blackmirror/controller/API/Websockets/MirrorApiWebsockets.kt

```diff
@@ -0,0 +1,118 @@
+package de.hhn.aib.labsw.blackmirror.controller.API.Websockets
+
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import okhttp3.*
+import java.io.IOException
+
+class MirrorApiWebsockets : WebSocketListener(),MirrorApi {
+    var sessions: ArrayList<WebSocket> = ArrayList()
+    private var mapper: ObjectMapper = ObjectMapper()
+    var listeners: HashMap<String, ArrayList<ApiListener>?> =
```

Comment:
perhaps use the more generic Map and List interfaces (doesn't kotlin have other collections anyway?)

Reply:
also applies to "sessions"